examples/otelcollector/README.md

```diff
@@ -0,0 +1,30 @@
+# Grafana Agent pull mode integration
+
+This example demonstrates how you can use Grafana Agent with Grafana Pyroscope to profile otel collector.
+
+### 1. Configure Grafana agent
+
+For Grafana Alloy to pull profiles from your application, configure the agent for things like profiling configuration, targets, and possibly authentication.
```

Comment:
Are you usingAlloy or Agent for this example? The links and text all point to Agent.